demo/contracts/hardhat.config.ts

```diff
@@ -30,9 +30,9 @@ const config: HardhatUserConfig = {
             accounts: [process.env.PRIVATE_KEY || ""],
         },
     },
-    etherscan: {
-        apiKey: process.env.ETHERSCAN_API_KEY,
-    },
+    // etherscan: {
```

Comment:
why?

Reply:
wasn't up to spend time on one more registration and key generation as value for this stage of development is absent

Reply:
you can leave this as it and simple do not generate key :) it will work

Reply:
we can always provide it when needed